web/src/connectors/document-page-connector/document-page-connector.tsx

```diff
@@ -42,10 +42,12 @@ export const DocumentPageConnector = () => {
 
     const { dataSource: documentJobDataSource, cache: documentJobsCache } = useEntity<
         DocumentJob,
-        string
+        number
     >(documentJobsFetcher, documentJobsFilters);
 
-    const [selectedDocumentJobId, setSelectedDocumentJobId] = useState('');
+    const [selectedDocumentJobId, setSelectedDocumentJobId] = useState<number | undefined>(
+        undefined
```

Comment:
i suppose it's better to use `null` here instead of `undefined`
https://learn.javascript.ru/types#:~:text=%D0%9D%D0%BE%20%D1%82%D0%B0%D0%BA%20%D0%B4%D0%B5%D0%BB%D0%B0%D1%82%D1%8C%20%D0%BD%D0%B5%20%D1%80%D0%B5%D0%BA%D0%BE%D0%BC%D0%B5%D0%BD%D0%B4%D1%83%D0%B5%D1%82%D1%81%D1%8F.%20%D0%9E%D0%B1%D1%8B%D1%87%D0%BD%D0%BE%20null%20%D0%B8%D1%81%D0%BF%D0%BE%D0%BB%D1%8C%D0%B7%D1%83%D0%B5%D1%82%D1%81%D1%8F%20%D0%B4%D0%BB%D1%8F%20%D0%BF%D1%80%D0%B8%D1%81%D0%B2%D0%BE%D0%B5%D0%BD%D0%B8%D1%8F%20%D0%BF%D0%B5%D1%80%D0%B5%D0%BC%D0%B5%D0%BD%D0%BD%D0%BE%D0%B9%20%C2%AB%D0%BF%D1%83%D1%81%D1%82%D0%BE%D0%B3%D0%BE%C2%BB%20%D0%B8%D0%BB%D0%B8%20%C2%AB%D0%BD%D0%B5%D0%B8%D0%B7%D0%B2%D0%B5%D1%81%D1%82%D0%BD%D0%BE%D0%B3%D0%BE%C2%BB%20%D0%B7%D0%BD%D0%B0%D1%87%D0%B5%D0%BD%D0%B8%D1%8F%2C%20%D0%B0%20undefined%20%E2%80%93%20%D0%B4%D0%BB%D1%8F%20%D0%BF%D1%80%D0%BE%D0%B2%D0%B5%D1%80%D0%BE%D0%BA%2C%20%D0%B1%D1%8B%D0%BB%D0%B0%20%D0%BB%D0%B8%20%D0%BF%D0%B5%D1%80%D0%B5%D0%BC%D0%B5%D0%BD%D0%BD%D0%B0%D1%8F%20%D0%BD%D0%B0%D0%B7%D0%BD%D0%B0%D1%87%D0%B5%D0%BD%D0%B0.